powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/CallbackUrlEntity.java

```diff
@@ -57,6 +58,9 @@
     @Convert(converter = CallbackAttributeConverter.class)
     private List<String> attributes;
 
+    @Column(name = "authentication")
```

Comment:
I think we should somehow define the `CallbackAuthenticationConverter` here, right?

Reply:
Indeed.